docs/ThirdPartyHosting.md

```diff
@@ -75,7 +82,36 @@ Specifically, third-party hosts should not enable the following policies without
 
 Third-party users who are granted access to an internal lightning node or hot wallet functionality to enable features such as Payjoin, should understand the risk and trust associated with [using hot wallets](./HotWallet.md) before choosing to use it. Use one of the [recommended wallets](./WalletSetup.md) which provide an xpubkey to use in your store, if you are unsure which wallet type to use.
 
+## Third Party Hosting FAQ
+
+## Where is the list of BTCPay third-party hosts?
+
+Feel free to chat with the [Community](./Community.md) to find the appropriate host for your needs, but also make sure to choose one that is trustworthy. Read the rest of this document to better understand the pros and cons of using a third-party host.
+
+The BTCPay Server [Directory](https://directory.btcpayserver.org/filter/hosts) lists multiple free or paid third-party hosts that you can register to, to start exploring BTCPay Server.
+
+## How can one become a third-party host?
+To become a third-party host, you need to self-host a BTCPay Server and enable registration for other users.
+Go to Server Settings > Policies > Disable registration, unmark the checkbox. You may also want to configure the [SMTP settings](./FAQ/FAQ-ServerSettings.md#how-to-configure-smtp-settings-in-btcpay) to allow them to reset their password if they forget it.
+
+## Are there any limitations in features when using a third-party host?
+Yes. Here are some restrictions.
+* No [Lightning Network](./LightningNetwork.md) by default. Can be enabled by the third-party host but at the cost of a security risk for registrants using that third-party. [More information](#can-i-enable-the-use-of-my-lightning-network-node-to-others)
+* No [wallet re-scan](./FAQ/FAQ-Wallet.md#what-is-wallet-re-scan-in-btcpay)
+* No [Server Settings](./Walkthrough.md#server-settings) access
+
+The limitations happen for technical reasons, mostly because these features require a user to run a full node to use them.
+
 ## Can I enable the use of my Lightning Network node to others?
 
 Yes, you can enable users that have registered on your BTCPay Server instance to use your Lightning Network node.
 [See how](./FAQ/FAQ-LightningNetwork.md#how-many-users-can-use-lightning-network-in-btcpay)
+
+## What does the third-party host know about their users?
```

Comment:
Not sure how I'd feel about this, since if malicious they can know way beyond this, right?